docs/jwst/stpipe/user_step.rst

```diff
@@ -101,24 +101,51 @@ class, pass the ``-h`` parameter, and the name of a Step class or
 parameter file::
 
     $ strun -h do_cleanup.asdf
-    usage: strun [--logcfg LOGCFG] cfg_file_or_class [-h] [--pre_hooks]
-                 [--post_hooks] [--skip] [--scale] [--extname]
-
-    optional arguments:
-      -h, --help       show this help message and exit
-      --logcfg LOGCFG  The logging configuration file to load
-      --verbose, -v    Turn on all logging messages
-      --debug          When an exception occurs, invoke the Python debugger, pdb
-      --pre_hooks
-      --post_hooks
-      --skip           Skip this step
-      --scale          A scale factor
-      --threshold      The threshold below which to apply cleanup
-      --output_file    File to save the output to
-
-Every step has an `--output_file` parameter.  If one is not provided,
-the output filename is determined based on the input file by appending
-the name of the step.  For example, in this case, `foo.fits` is output
+    usage: strun [-h] [--debug] [--save-parameters SAVE_PARAMETERS] [--disable-crds-steppars] [--verbose] [--log_level LOG_LEVEL] [--log_format LOG_FORMAT] [--log_file LOG_FILE] [--log_stream LOG_STREAM]
```

Comment:
BTW rather than hardcoding this and risk it getting out of sync with reality, you could do dynamic output injection like this:

* https://github.com/spacetelescope/jdaviz/pull/3286